department-of-veterans-affairs/caseflow
# Add DVC team & DVC team modal to team management page

Part 3 of stack
Part 1 #14981 
Part 2 #14982 

### Description
This PR adds the frontend additions for the new DVC team including the changes to the team management page and the DVC modal.

### Acceptance Criteria
- [x] Can successfully add a new DVC team and add/remove members from the team

### Testing Plan
1. Sign in as TEAM_ADMIN
2. Go to http://localhost:3000/team_management
3. Make sure the DVC teams header and +Add DVC Team button are present
4. Click on Add DVC Team and make sure modal appears
5. Select a user to create a DVC team for (remember the css_id you choose) and hit Submit
6. Ensure the Success banner appears and a new team is listed under DVC teams
7. Go to the Org admin page link and make sure you can add & remove team members

### User Facing Changes

#### BEFORE
<img width="1234" alt="Screen Shot 2020-08-14 at 3 38 47 PM" src="https://user-images.githubusercontent.com/18618189/90286658-4ed5d380-de44-11ea-99c5-e89e73b3c5ed.png">

#### AFTER
<img width="1239" alt="Screen Shot 2020-08-14 at 3 38 29 PM" src="https://user-images.githubusercontent.com/18618189/90286665-5301f100-de44-11ea-869e-77f8c37a4297.png">

#### AFTER
<img width="1255" alt="Screen Shot 2020-08-19 at 6 29 21 PM" src="https://user-images.githubusercontent.com/18618189/90696286-fb9bc080-e249-11ea-8bbf-131ede93cd16.png">
